test/functional/uvm_vsmb_test.go

```diff
@@ -38,3 +42,42 @@ func TestVSMB(t *testing.T) {
 }
 
 // TODO: VSMB for mapped directories
+
+func TestVSMB_Writable(t *testing.T) {
+	testutilities.RequiresBuild(t, osversion.RS5)
+
+	opts := uvm.NewDefaultOptionsWCOW(t.Name(), "")
+	opts.NoWritableFileShares = true
+	// uvmScratchDir := testutilities.CreateTempDir(t)
+	// opts.LayerFolders = []string{"C:\\ContainerPlatData\\root\\io.containerd.snapshotter.v1.windows\\snapshots\\60", uvmScratchDir}
+	// vm := testutilities.CreateWCOWUVMFromOpts(context.Background(), t, opts)
```

Comment:
Looks like comments from testing things that can go

Reply:
I believe @helsaawy will be making a separate PR to address this.

Reply:
I am updating the tests now, since I had to hard code the paths to work

Reply:
I changed the functional tests back to their original form: they do not run and need updates to the functional tests harness, which I moved to this branch: https://github.com/helsaawy/hcsshim/tree/he/functests